src/lib/worker.js

```diff
@@ -325,6 +333,39 @@ var getPodStableNetworkAddressAndPort = function (pod) {
   );
 };
 
+var hasSuffix = function (member) {
+  // console.log("🚀 ~ file: worker.js:341 ~ hasSuffix ~ member:", member)
+  suffix =     (config.k8sMongoServiceName + "." +
+  config.namespace +
+  ".svc." +
+  config.k8sClusterDomain +
+  ":" +
+  config.mongoPort)
+  if (member.name) {
+    return member.name.endsWith(suffix)
+  }
+  return true //during initial stage we might get member name as empty so return true
+}
+
+function isIPAddress(ipAddress) {
```

Comment:
https://stackoverflow.com/questions/4460586/javascript-regular-expression-to-check-for-ip-addresses
This might check better?

Reply:
would revisit them as of now not using it